azkaban-common/src/main/java/azkaban/executor/ExecutableFlow.java

```diff
@@ -263,7 +263,7 @@ public void setFlowLockErrorMessage(final String flowLockErrorMessage) {
     flowObj.put(SUBMITTIME_PARAM, this.submitTime);
 
     final List<Map<String, Object>> slaOptions = new ArrayList<>();
-    this.executionOptions.getSlaOptions().stream()
+    this.executionOptions.getSlaOptions()
```

Comment:
This code can be changed as mentioned in one of the comment.

Reply:
Will do